src/components/Quotes.js

```diff
@@ -0,0 +1,64 @@
+/** ****************************************
+ * Project: math-magicians
+ * File: Quotes.js
+ * Created: 3/30/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ ****************************************** */
+import React, { useEffect, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+import LoadingSpinner from './LoadingSpinner';
+
+export default function Quotes() {
+  const [quote, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleFetch = async () => {
+    const $url = 'https://api.api-ninjas.com/v1/quotes';
+    setIsLoading(true);
+    await fetch($url,
+      {
+        headers: {
+          'X-Api-Key': 'VB6fCHNwu864O21lpthw6IV8hNhobYyC8207wMed',
+        },
+      })
+      .then((respose) => respose.json())
+      .then((respose) => {
+        setQuote(respose);
+        setIsLoading(false);
+        // Optional code to simulate delay
+        /* setTimeout(() => {
+          setQuote(respose);
+          setIsLoading(false);
+        }, 5000); */
```

Comment:
- [ ] [OPTIONAL]
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.